tests/Settings.json

```diff
@@ -1,19 +1,21 @@
 {
-  "debug": true,
-  "debug_json": true,
-  "production": false,
-  "arr": [1, 2, 3, 4, 5, 6, 7, 8, 9, 10],
-  "place": {
-    "name": "Torre di Pisa",
-    "longitude": 43.7224985,
-    "latitude": 10.3970522,
-    "favorite": false,
-    "reviews": 3866,
-    "rating": 4.5,
-    "creator": {
-      "name": "John Smith",
-      "username": "jsmith",
-      "email": "jsmith@localhost"
-    }
-  }
+    "debug": true,
+    "debug_json": true,
+    "production": false,
+    "arr": [1, 2, 3, 4, 5, 6, 7, 8, 9, 10],
+    "place": {
+        "name": "Torre di Pisa",
+        "longitude": 43.7224985,
+        "latitude": 10.3970522,
+        "favorite": false,
+        "reviews": 3866,
+        "rating": 4.5,
+        "creator": {
+            "name": "John Smith",
+            "username": "jsmith",
+            "email": "jsmith@localhost"
+        }
+    },
```

Comment:
AFAICS you only changed the indentation here. Care to revert this so we get a smaller diff?

Reply:
there two more line that I have added to this file 
` "FOO": "FOO should be overridden",
    "bar": "I am bar"
 `
the extra space is introduced by running rustfmt

Reply:
rustfmt changes a JSON file?

Reply:
Sorry I think that has to do with the extension I am using when I hit save, my bad.